library/oneview_certificates_server_facts.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2020) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_certificates_server_facts
+short_description: Retrieve the facts about one or more of the OneView Server Certificates
+description:
+    - Retrieve the facts about one or more of the Server Certificates from OneView.
+version_added: "2.4"
+requirements:
+    - "python >= 3.4.2"
+    - hpOneView >= 5.1.0
+author: "Venkatesh Ravula (@VenkateshRavula)"
+options:
+    alias_name:
+      description:
+        - Server Certificate aliasname.
+
+extends_documentation_fragment:
+    - oneview
+    - oneview.factsparams
+'''
+
+EXAMPLES = '''
+- name: Gather facts about a Server Certificate by remote address
+  oneview_certificates_server_facts:
+    config: "{{ config }}"
```

Comment:
You need to replace config with the actual parameter values. It should be a best practice overall to keep our documentation synced with the modules in the ansible core repository.